src/client/app/components/AdminComponent.jsx

```diff
@@ -46,58 +77,95 @@ export default class AdminComponent extends React.Component {
+									type="text"
+									placeholder="Name"
+									value={this.props.displayTitle}
+									onChange={this.handleDisplayTitleChange}
+									maxLength={50}
+								/>
 							</div>
-							<div className="radio">
-								<label>
-									<input
-										type="radio"
-										name="chartTypes"
-										value={chartTypes.bar}
-										onChange={this.handleDefaultChartToRenderChange}
-										checked={this.props.defaultChartToRender === chartTypes.bar}
-									/>
-									Bar
-								</label>
+							<div>
+								<p style={labelStyle}>Default Graph Type:</p>
+								<div className="radio">
+									<label>
+										<input
+											type="radio"
+											name="chartTypes"
+											value={chartTypes.line}
+											onChange={this.handleDefaultChartToRenderChange}
+											checked={this.props.defaultChartToRender === chartTypes.line}
+										/>
+                                        Line
+                                    </label>
+								</div>
+								<div className="radio">
+									<label>
+										<input
+											type="radio"
+											name="chartTypes"
+											value={chartTypes.bar}
+											onChange={this.handleDefaultChartToRenderChange}
+											checked={this.props.defaultChartToRender === chartTypes.bar}
+										/>
+                                        Bar
+                                    </label>
+								</div>
+								<div className="radio">
+									<label>
+										<input
+											type="radio"
+											name="chartTypes"
+											value={chartTypes.compare}
+											onChange={this.handleDefaultChartToRenderChange}
+											checked={this.props.defaultChartToRender === chartTypes.compare}
+										/>
+                                        Compare
+                                    </label>
+								</div>
 							</div>
-							<div className="radio">
+							<div className="checkbox">
 								<label>
 									<input
-										type="radio"
-										name="chartTypes"
-										value={chartTypes.compare}
-										onChange={this.handleDefaultChartToRenderChange}
-										checked={this.props.defaultChartToRender === chartTypes.compare}
+										type="checkbox"
+										onChange={this.handleDefaultBarStackingChange}
+										checked={this.props.defaultBarStacking}
 									/>
-									Compare
+									Default Bar stacking
 								</label>
 							</div>
+							<Button
+								type="submit"
+								onClick={this.handleSubmitPreferences}
+								disabled={this.props.disableSubmitPreferences}
+							>
+								Submit
+							</Button>
 						</div>
-						<div className="checkbox">
-							<label><input type="checkbox" onChange={this.handleDefaultBarStackingChange} checked={this.props.defaultBarStacking} />Default Bar stacking</label>
+					</div>
+					<div className="row">
+						<div className="col-xs-2">
+							<Dropzone accept="text/csv, application/vnd.ms-excel," onDrop={this.handleFileToImport}>
```

Comment:
Strongly recommend that you do not show the Dropzone unless a meter has been selected. I recommend moving the whole dropzone + meter select into a new component, then rendering that component here.